Plugins/ExternalTexture/Source/ExternalTexture_Shared.h

```diff
@@ -16,14 +16,9 @@ namespace Babylon::Plugins
 
     void ExternalTexture::Impl::Update(Graphics::TextureT ptr)
     {
-        Info info;
-        GetInfo(ptr, info);
-        if (info != m_info)
-        {
-            throw std::runtime_error{"Textures must have same info"};
-        }
```

Comment:
This comparison is flawed. It is okay to update even if the info is not the same.

Reply:
There is a missing update to `m_info` here.